ts/src/bitget.ts

```diff
@@ -1116,7 +1139,14 @@ export default class bitget extends Exchange {
         let inverse = undefined;
         let expiry = undefined;
         let expiryDatetime = undefined;
-        if (typeId === 'SPBL') {
+        let margin = false;
+        if (typeId === undefined) {
+            type = 'margin';
+            margin = true;
+            contract = true;
+            pricePrecision = this.parseNumber (this.parsePrecision (this.safeString (market, 'priceScale')));
+            amountPrecision = this.parseNumber (this.parsePrecision (this.safeString (market, 'quantityScale')));
```

Comment:
currently, the regular spot symbol is the same as the spot margin, so the margin values will override the spot ones right? I'm not sure we can do that